src/main/scala/akka/persistence/cassandra/query/EventsByTagPublisher.scala

```diff
@@ -60,6 +63,14 @@ private[query] class EventsByTagPublisher(
     case None       => Long.MaxValue
   }
 
+  // Subscribe to DistributedPubSub so we can receive immediate notifications when the journal has written something.
+  if (eventualConsistencyDelayMillis == 0) {
```

Comment:
I think it would be better to use the `pubsub-minimum-interval` here to disable subscription. Then we have one setting that controls if this feature is enabled or not. Someone might want to run with a long (e.g. 30s) poll interval and still use this feature with eventualConsistencyDelay != 0 (e.g. 10s), but then we should delay the `Continue` message in `case tagWritten`, i.e. 

```
   case tagWritten: String if tagWritten == tag =>
     if (eventualConsistencyDelayMillis == 0) self ! Continue
     else context.system.scheduler.scheduleOnce(eventualConsistencyDelay, self, Continue)
```

WDYT?